### DISCUSSION TITLE
How to upgrade to v10?

### DISCUSSION THREAD

**Discussion by bstro** [vercel/next.js#19090] [Help] [answered]
Hi there, very new to Next.js and feeling like I'm missing something (I've done a considerable amount of googling/digging through Github) — I'm wondering how to upgrade from Next.js 9.x to the latest release. There appears to be an "Upgrade Guide" in the official docs, but nothing on how to upgrade to version 10. Is it simply bumping the version number in package.json? Do I need to upgrade any other versions? Can I upgrade React to 17.x as well?

**Comment by kachkaev** [comment, marked as answer]
> simply bumping the version number in package.json

Yep, that's the right direction. You start with manually tweaking the version in `package.json`, then open a terminal and run `npm install` (or `yarn install` if you are using Yarn instead on NPM).

> Can I upgrade React to 17.x as well?

Yep, you can if you want to. The process is the same: to tweak the version in package.json first, then run `npm install`.

**Reply by bstro** [reply]
thx!

**Comment by timneutkens** [comment]
I've updated the upgrading guide here: https://github.com/vercel/next.js/pull/19101
